### PR TITLE
Add line mentioning dev dependencies

### PR DESCRIPTION
**Pull request by bengineer19**
A very small change, but something which might save someone a few mins of head scratching

### CODE REVIEW

**Comment by tuxxy** [comment]
Hey, thanks for the contribution!

More specifically, what problem did you encounter to make this change? By development dependencies, do you mean something like `python3-dev`?

We'd like to get as much information as possible so our users can avoid needless errors or problems during installation/usage. :D

**Comment by bengineer19** [comment]
All I wanted to make clear was that the [development dependencies outlined in the installation guide](https://nucypher.readthedocs.io/en/latest/guides/installation_guide.html#development-installation) need to be installed to run a demo :)

File: docs/source/demos/local_fleet_demo.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ Running A Local Fleet
 
 1. Install Nucypher
 
-Acquire the Nucypher application code and install the dependencies.
+Acquire the Nucypher application code and install the dependencies. Ensure the development dependencies are installed.
```
Comment (KPrasch) [inline docs/source/demos/local_fleet_demo.rst]:
```suggestion
Acquire the NuCypher application code and install the development dependencies.
```

Comment (KPrasch) [reply docs/source/demos/local_fleet_demo.rst]: Heh - I surmise you are running one of our demos? :-)

**Comment by tuxxy** [comment]
@bengineer19 If you don't have any holdups with @KPrasch's suggestion, you can go ahead and apply it and I'll merge your contribution. :)

**Comment by KPrasch** [comment]
Hello again! Just checking in - I'd like to know which error you encountered.  Which development dependency was required for you to run the demos?  We intend for demos to be usable *without* a development installation.

Thanks again!

**Comment by bengineer19** [comment]
That makes sense.
It was a few weeks ago that it wasn't working for me. I seem to remember installing the dev dependencies and everything going smoothly after that, but it's so long ago now that it's clouded by the mists of time.
I tried to recreate the problem on a different machine just now and was unable to - so perhaps it was a reboot/something else that fixed it before.
Sorry; for now it seems like my comments were unfounded!

**Pull request closed without merging**